IsraelHiking.Web/src/application/services/route-statistics.service.ts

```diff
@@ -107,7 +107,10 @@ export class RouteStatisticsService {
 
         // smooth the line in order to better calculate gain and loss:
         // changing x from Km to Km * 100 to better align with required altitude sensitivity
-        let simplifiedCoordinates = SpatialService.simplify(routeStatistics.points.map(p => [p.coordinate[0] * 100, p.coordinate[1]]), 5);
+        let pts = resample(routeStatistics.points.map(p=>p.coordinate), { dist: 0.025 }, false);
+        var createMedianFilter = require('moving-median')
```

Comment:
Better to `import` it at the top like you did with the `resample` (it's a bit different due to how it is exposed in the library that you are using, it's extremely annoying and I've waisted a lot of time understanding this).
You'll probably need to write `import createMedianFilter from "moving-median";`